osu.Game/Overlays/BeatmapSet/Scores/TopScoreUserSection.cs

```diff
@@ -116,6 +116,7 @@ public ScoreInfo Score
         {
             set
             {
+                avatar.Reset();
```

Comment:
Could setting `TransformImmediately` to true on `UpdateableAvatar` work better? Should do all of this internally.

Reply:
That requires overriding both TransformImmediately and TransformDuration, otherwise the fade animation would still play while setting the Model to null. I've done that now.